packages/language-server/test/completion-items-spec.ts

```diff
@@ -160,6 +169,7 @@ describe("the UI5 language assistant Code Completion Services", () => {
     const suggestionNames = map(suggestions, suggestion => ({
```

Comment:
this is no longer `suggestionNames`

Reply:
Chaned to `suggestionsDetails` in all tests